internal/mode/static/nginx/config/split_clients_test.go

```diff
@@ -97,28 +97,15 @@ func TestExecuteSplitClients(t *testing.T) {
 	}
 
 	for _, test := range tests {
+		g := NewWithT(t)
 		sc := string(executeSplitClients(dataplane.Configuration{BackendGroups: test.backendGroups}))
 
 		for _, expSubString := range test.expStrings {
-			if !strings.Contains(sc, expSubString) {
-				t.Errorf(
-					"executeSplitClients() did not generate split clients with substring %q for test %q. Got: %v",
-					expSubString,
-					test.msg,
-					sc,
-				)
-			}
+			g.Expect(strings.Contains(sc, expSubString)).To(BeTrue())
```

Comment:
```suggestion
			g.Expect(sc).To(ContainSubstring(expSubString))
```

Can do this for other instances like this.